### ISSUE TITLE
[BUG] `secureTextEntry` prop declared and not used

### ISSUE THREAD

**Issue by erheron** [Cnilton/react-native-floating-label-input#95]
In the [definition](https://github.com/Cnilton/react-native-floating-label-input/blob/7e2fa051b58127445bbc5c61bb9b1fea00f2c53a/src/index.tsx#L30) of `Props` type we have `Props extends TextInputProps` and because `TextInputProps` declares `secureTextEntry`, then `Props` type has `secureTextEntry` as well. 

However, quick look through the code is enough to say certainly that if `secureTextEntry` prop is unused.
I think it's misleading to users of this library, because they are allowed to pass `secureTextEntry` but it won't work.

Either we should use `secureTextEntry` in code (get it from props and pass to react-native's `TextInput` component) or we should remove it from the `Props` type, e.g. by modifying code like this 👇🏻 
```
export interface Props extends TextInputProps --> // modify this to be as below
export interface Props extends Omit<TextInputProps, "secureTextEntry">
```

**Comment by Cnilton** [comment]
Hi @erheron, that sounds good, would you mind creating a pull request fixing that?

**Comment by erheron** [comment]
Sure @Cnilton , I'll open a PR soon

**Comment by erheron** [comment]
Closed by #99
